src/OpenLoco/Objects/ObjectManager.cpp

```diff
@@ -1034,12 +1035,15 @@ namespace OpenLoco::ObjectManager
         for (const auto& header : objects)
         {
             auto id = getObjectId(index);
-            if (!load(header, id))
+            if (!header.isEmpty())
             {
-                result.success = false;
-                result.problemObject = header;
-                unloadAll();
-                break;
+                if (!load(header, id))
```

Comment:
not important, but you can merge these `if`s together:
`if (!header.isEmpty() && !load(header, id))`

Reply:
I'll leave it up to you if you want to change it, but I'll approve this since it doesn't really make a difference, it's just code style